config/brokers/mt-channel-broker/deployments/controller.yaml

```diff
@@ -95,3 +101,25 @@ spec:
           containerPort: 9090
         - name: profiling
           containerPort: 8008
+---
```

Comment:
not sure we want this below, at least not in this PR

Reply:
We should have it, especially for gathering metrics via ServiceMonitors

Reply:
Can you then update this PR description (and release notes) stating that these controllers are now exposed?

Reply:
aren't ServiceMonitors running within the cluster?

Reply:
I'd opt to move this to a different PR give the nature of the described changes.  Thanks!

Reply:
Updated release notes.

> aren't ServiceMonitors running in the cluster?

Yes

Reply:
+1 on moving that to a new PR

/cc @skonto regarding `ServiceMonitors`

Reply:
I'll move the new Services to a separate PR. 

FYI, Service Monitors are CRDs that ship with Prometheus Operator. It is a way of defining Prometheus scrape configuration.